frontend/src/components/core/Input.tsx

```diff
@@ -30,49 +35,77 @@ export const PureTextInputTitle = styled.input.attrs(() => ({
 export const LargeCenterInputText = styled(Input).attrs(() => ({
   type: 'text',
 }))`
-  width: 20rem;
+  width: ${({ width }) => width || '16rem'};
+  height: ${({ height }) => height || '3rem'};
   text-align: center;
-  font-size: ${({ theme }) => theme.fontSize.xMediumLarge};
-  padding: 1rem;
+  font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+  font-family: ${({ theme }) => theme.font};
+  font-weight: normal;
   color: ${({ theme }) => theme.colors.text};
-
-  &:focus {
-    border: 3px solid ${({ theme }) => theme.colors.darkBlue};
-  }
+  padding: 1rem;
 `;
 
-export const LargeInputButton = styled(Input).attrs(() => ({
+export const PrimaryInput = styled(Input).attrs(() => ({
   type: 'button',
 }))`
-  width: 20rem;
-  font-size: ${({ theme }) => theme.fontSize.xMediumLarge};
-  padding: 1rem;
-  color: white;
-  background-color: ${({ theme }) => theme.colors.blue};
-  font-weight: 700;
+  font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+  background: ${({ theme }) => theme.colors.gradients.blue};
+  color: ${({ theme }) => theme.colors.white};
+  width: ${({ width }) => width || '16rem'};
+  height: ${({ height }) => height || '3rem'};
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.24);
+  border: none !important;
+  font-weight: 400;
+  min-width: 150px;
+  min-height: 40px;
   cursor: pointer;
 
+  &:hover {
+    cursor: pointer;
+    box-shadow: 0 2px 12px rgba(0, 0, 0, 0.24);
+  }
+  
   &:disabled {
-    background-color: ${({ theme }) => theme.colors.lightBlue};
-    border: 3px solid ${({ theme }) => theme.colors.lightBlue};
-    cursor: default;
+    background: ${({ theme }) => theme.colors.gray};
+
+    &:hover {
+      cursor: default;
+      box-shadow: 0 1px 8px rgba(0, 0, 0, 0.24);
+    }
   }
 `;
 
 export const ConsoleTextArea = styled.textarea`
   font-family: Monaco, monospace;
   font-size: ${({ theme }) => theme.fontSize.default};
-  color: ${({ theme }) => theme.colors.text};
+  color: ${({ theme }) => theme.colors.darkText};
   margin: 2px;
+  padding: 8px;
+  
   min-width: 50px;
   max-width: 90%;
-  width: 90%;
-  min-height: 24px;
+  width: 50%;
+  min-height: 22px;
+  height: unset;
   max-height: 150px;
+  white-space: pre;
+  resize: both;
+  
+  /* Hide scrollbar for Chrome, Safari and Opera */
```

Comment:
How do you test all of this - is there a way to check cross-browser functionality quickly?

Reply:
This comment is from the post that gave the solution code - as for testing, I don't have a quick method besides just doing the manual testing that we usually do on Chrome/Safari/Firefox. For clarity though I'll remove the Opera from this comment